packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/portal/PortalControllerIntegrationTests.java

```diff
@@ -67,6 +73,8 @@ public class PortalControllerIntegrationTests extends AbstractIntegrationTests {
 
     private static final String SAMPLE_PORTAL = "Sample Portal";
 
+    private static final String MISSING_PORTAL = "Missing Portal";
```

Comment:
Don't create constant like that. Being able to read test easily is way more important than creating such constant. There is not maintainability issue with having some inline text. On the other hand, having to check the real value of a constant is painful especially in tests.